src/ir/value.h

```diff
@@ -101,10 +101,11 @@ class Any {
 // A class that represents a data value.
 class Value {
  public:
-  Value(value::BlockArgument arg) : value_(std::move(arg)) {}
-  Value(value::OperationResult arg) : value_(std::move(arg)) {}
-  Value(value::Any arg) : value_(std::move(arg)) {}
-  Value(value::StoredValue arg) : value_(std::move(arg)) {}
+  using ValueVariants =
```

Comment:
Could simply be `Variants` as in `Value::Variants`.

Reply:
Done.